example/variables.tf

```diff
@@ -1,5 +1,23 @@
 # Required
 
+variable "log_api_gateway_to_cloudwatch" {
```

Comment:
I don't really like having to define 3 more variables that users will have to figure out if/how they want to configure. That being said, this configuration was built as *opt in* so a user by default doesn't need them.